client/http/http.go

```diff
@@ -19,13 +19,17 @@ import (
 	"github.com/beatlabs/patron/log"
 	"github.com/beatlabs/patron/reliability/circuitbreaker"
 	"github.com/beatlabs/patron/trace"
+	wct "github.com/weaveworks/common/tracing"
```

Comment:
We are using only a small function. Would it not be better to add this function to our trace package and avoid a dependency?

Reply:
OpenTracing has updated the spec to add [`SpanContext.ToTraceID`](https://github.com/opentracing/specification/blob/master/rfc/trace_identifiers.md#specification-changes) but this is not available in the Go implementation yet.
See also: https://github.com/opentracing/specification/issues/24